pkg/controller/discovery/container/hook.go

```diff
@@ -0,0 +1,138 @@
+package container
+
+import (
+	"context"
+	"time"
+
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	"github.com/konveyor/mig-controller/pkg/controller/discovery/model"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+//
+// A collection of k8s MigHook resources.
+type MigHook struct {
```

Comment:
Hmm, this is the lone instance where *Mig* snuck into the name of the struct in discovery DB. Goes against convention. I will fix.